core/templates/pages/signup-page/signup-page.component.html

```diff
@@ -41,7 +41,7 @@
           </div>
         </div>
 
-        <div *ngIf="showEmailPreferencesForm" class="form-group row">
```

Comment:
@leolindem Is showEmailPreferencesForm still needed in the codebase? If not, could you please delete it in general from the code and tests? Thanks!